Modules/System/SharePoint/src/model/list/SharePointList.Table.al

```diff
@@ -0,0 +1,94 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+/// <summary>
+/// Holds information about list entity.
+/// </summary>
+table 9105 "SharePoint List"
+{
+    Access = Public;
+    DataClassification = SystemMetadata;
+    Caption = 'SharePoint List';
+    TableType = Temporary;
+    Extensible = false;
+
+    fields
+    {
+        field(1; Id; Guid)
+        {
+            DataClassification = SystemMetadata;
+            Caption = 'Id';
+        }
+
+        field(2; Title; Text[250])
+        {
+            DataClassification = SystemMetadata;
```

Comment:
Most if not all of these text fields are populated from SharePoint which the customer uploaded/set? If so, the dataclassification should be "CustomerContent".

Reply:
Hi,

Yes, everything is 1:1 with SharePoint entities.
I used to have CustomerContent everywhere but was told to reconsider :), see:

https://github.com/microsoft/ALAppExtensions/pull/18635#discussion_r913700758

Reply:
Switched to CustomerContnet